NewApp/ngsa-csharp/Ngsa.App/Controllers/GenresController.cs

```diff
@@ -14,17 +14,14 @@ namespace Ngsa.App.Controllers
     [Route("api/[controller]")]
     public class GenresController : Controller
     {
-        private readonly ILogger logger;
-
-        /// <summary>
-        /// Constructor
-        /// </summary>
-        /// <param name="logger">log instance</param>
-        /// <param name="dal">data access layer instance</param>
-        public GenresController(ILogger<GenresController> logger)
+        private static readonly NgsaLog Logger = new NgsaLog
         {
-            this.logger = logger;
-        }
+            Name = typeof(GenresController).FullName,
+            LogLevel = App.AppLogLevel,
+            ErrorMessage = "GenresControllerException",
+            NotFoundError = "Movie Not Found",
+            Method = nameof(GetGenresAsync),
```

Comment:
I don't think this will ever return a NotFoundError, but if required, should be "Genre Not Found"

Reply:
Good catch - fixed